sdk/digitaltwins/Azure.DigitalTwins.Core/CHANGELOG.md

```diff
@@ -20,3 +20,4 @@
 
 ## 1.0.0-preview.2
```

Comment:
Shouldn't the newest releases be on top?

Reply:
I think that would look better as well but every instance of this I have seen in the repo looks like this.